From c2512196ae301c7dbc71dab0083b4dd563b3c1e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Tue, 7 Jun 2011 15:03:15 +0800
Subject: [PATCH] Revert "Revert "rk29: clock: always enable tpiu clock for
 jtag debug""

This reverts commit 6aca2b5c5ccb05ae0c02e23ec4519e66e5af4ea6.
---
 arch/arm/mach-rk29/clock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-rk29/clock.c b/arch/arm/mach-rk29/clock.c
index 12705ac78ce4..47bd254caa58 100755
--- a/arch/arm/mach-rk29/clock.c
+++ b/arch/arm/mach-rk29/clock.c
@@ -2476,6 +2476,7 @@ static void __init clk_enable_init_clocks(void)
 	clk_enable_nolock(&clk_intmem);
 	clk_enable_nolock(&clk_ddr);
 	clk_enable_nolock(&clk_debug);
+	clk_enable_nolock(&clk_tpiu);
 	clk_enable_nolock(&clk_jtag);
 	clk_enable_nolock(&clk_uart1);
 }
-- 
2.35.3

